# Central and Local Government Unregistered Land

-Ministry of Housing, Communities and Local Government 
Last updated: 06 August 2020

A list of central and local government land in England, which may not be registered with HM Land Registry (HMLR).

HMLR has created this dataset for the Ministry for Housing, Communities and Local Government (MHCLG) by combining HMLR freehold polygon data with the public sector ownership data currently openly available from the Office of Government Property.

The following fields comprise the address data included in the dataset

Property Name,
Street No,
Road,
Town,
Postcode

In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [5]:
df = pd. read_csv('ul.csv')

In [6]:
df.head()

Unique_Reference_Number                       Property_Name  \
0                   119866  JAFFRAY BEHAVIOURAL SUPPORT CENTRE   
1                   138624      UNIT 1A UNITED DOWNS IND. EST.   
2                   138625      UNIT 2A UNITED DOWNS IND. EST.   
3                   138626      UNIT 1B UNITED DOWNS IND. EST.   
4                   138627      UNIT 1C UNITED DOWNS IND. EST.   

                 Property_Centre         Department        Town Postcode  \
0  BIRMINGHAM CITY COUNCIL (BIR)  Local Authorities  BIRMINGHAM   B236AB   
1         CORNWALL COUNCIL (CON)  Local Authorities     REDRUTH  TR165HY   
2         CORNWALL COUNCIL (CON)  Local Authorities     REDRUTH  TR165HY   
3         CORNWALL COUNCIL (CON)  Local Authorities     REDRUTH  TR165HY   
4         CORNWALL COUNCIL (CON)  Local Authorities     REDRUTH  TR165HY   

   LandAreaHA  OSGBGridRefEast  OSGBGridRefNorth ONS_Administrative_Area  \
0      1.2774      410318.0938       291177.4063              Birmingham   
1      0.0100      174555.0000        41474.0000                Cornwall   
2      0.0100      174561.0000        41426.0000                Cornwall   
3      0.0100      174562.0000        41477.0000                Cornwall   
4      0.0100      174570.0000        41482.0000                Cornwall   

    ONS_CODE  
0  E08000025  
1  E06000052  
2  E06000052  
3  E06000052  
4  E06000052

In [7]:
df.shape


(16608, 11)

In [8]:
df['Town'].value_counts()

BIRMINGHAM      3281
LIVERPOOL        900
COVENTRY         571
BARNSLEY         466
ROCHDALE         390
                ... 
THAME              1
EGHAM              1
BELPER             1
SOUTH PETHER       1
WASHINGTON         1
Name: Town, Length: 733, dtype: int64

### INSIGHT 1:
1. Bimingham has the highest unlisted properties.
2. Liverpool and Coventry have the second and third highest number of unlisted properties.

In [9]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 16608 entries, 0 to 16607
Data columns (total 11 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   Unique_Reference_Number  16608 non-null  int64  
 1   Property_Name            16608 non-null  object 
 2   Property_Centre          16608 non-null  object 
 3   Department               16608 non-null  object 
 4   Town                     16608 non-null  object 
 5   Postcode                 16309 non-null  object 
 6   LandAreaHA               15322 non-null  float64
 7   OSGBGridRefEast          16608 non-null  float64
 8   OSGBGridRefNorth         16608 non-null  float64
 9   ONS_Administrative_Area  16608 non-null  object 
 10  ONS_CODE                 16608 non-null  object 
dtypes: float64(3), int64(1), object(7)
memory usage: 1.4+ MB


In [11]:
df['Property_Centre'].value_counts()

BIRMINGHAM CITY COUNCIL (BIR)                       3301
LIVERPOOL CITY COUNCIL (LIV)                         894
COVENTRY CITY COUNCIL (COV)                          569
HAMPSHIRE COUNTY COUNCIL (HAM)                       524
BARNSLEY METROPOLITAN BOROUGH COUNCIL (BNS)          502
                                                    ... 
LONDON BOROUGH OF WALTHAM FOREST (WFT)                 1
NORTH WEST LEICESTERSHIRE DISTRICT COUNCIL (NWL)       1
MOJ - HMCTS (MIDLANDS)                                 1
DEFRA - ENVIRONMENT AGENCY SOUTH WEST                  1
LONDON BOROUGH OF HOUNSLOW (HNS)                       1
Name: Property_Centre, Length: 252, dtype: int64

### INSIGHT 2:
Out of 3281 properties in Birmingham, Birmingham City Council owns 3301 properties.


In [20]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 16608 entries, 0 to 16607
Data columns (total 11 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   Unique_Reference_Number  16608 non-null  int64  
 1   Property_Name            16608 non-null  object 
 2   Property_Centre          16608 non-null  object 
 3   Department               16608 non-null  object 
 4   Town                     16608 non-null  object 
 5   Postcode                 16309 non-null  object 
 6   LandAreaHA               15322 non-null  float64
 7   OSGBGridRefEast          16608 non-null  float64
 8   OSGBGridRefNorth         16608 non-null  float64
 9   ONS_Administrative_Area  16608 non-null  object 
 10  ONS_CODE                 16608 non-null  object 
dtypes: float64(3), int64(1), object(7)
memory usage: 1.4+ MB


# INSIGHT 3:
There are several null values in the Land Area column. The data is incomplete.    

In [22]:
df.describe()

Unique_Reference_Number    LandAreaHA  OSGBGridRefEast  \
count             16608.000000  15322.000000     16608.000000   
mean             261610.104528     20.135247    424522.137738   
std              153398.734811    380.243934     71329.466615   
min               20853.000000      0.000000     90118.000000   
25%              158064.250000      0.010000    390573.750000   
50%              234893.500000      0.066350    410846.843750   
75%              326686.250000      0.545225    451496.000000   
max              899953.000000  26808.408200    654838.000000   

       OSGBGridRefNorth  
count      16608.000000  
mean      297498.912031  
std       114622.830447  
min        10384.000000  
25%       222960.500000  
50%       288889.000000  
75%       391245.500000  
max       653417.000000

# Insight 4:
The average area accross England is 20.135

# Insight 5:
Maximum area available is 26808.40820 which is available in Lancaster. 

In [30]:
df.loc[df['LandAreaHA']==26808.408200]

Unique_Reference_Number  \
10027                   261029   

                                           Property_Name  \
10027  QUERNMORE ROAD - HIGHFIELD RECREATION GROUND -...   

                    Property_Centre         Department       Town Postcode  \
10027  LANCASTER CITY COUNCIL (LAC)  Local Authorities  LANCASTER   LA13ES   

       LandAreaHA  OSGBGridRefEast  OSGBGridRefNorth ONS_Administrative_Area  \
10027  26808.4082         348678.0          461623.0               Lancaster   

        ONS_CODE  
10027  E07000121